grakn-test/src/test/java/ai/grakn/test/engine/backgroundtasks/ClusterManagerTest.java

```diff
@@ -20,14 +20,19 @@
 
 import ai.grakn.engine.backgroundtasks.distributed.Scheduler;
 import ai.grakn.engine.backgroundtasks.distributed.ClusterManager;
+import ai.grakn.test.EngineContext;
+import org.junit.ClassRule;
 import org.junit.Ignore;
 import org.junit.Test;
 
 import static org.junit.Assert.assertNotEquals;
 
-public class ClusterManagerTest { //extends EngineTestBase {
+public class ClusterManagerTest {
     private final ClusterManager clusterManager = ClusterManager.getInstance();
 
+//    @ClassRule
+//    public static final EngineContext engine = EngineContext.startServer();
+
```

Comment:
Because all the tests in this class are ignored, we didn't want to waste time purposelessly starting and stopping engine :( It's on the to-do list to fix this test this week, though